config.toml

```diff
@@ -27,7 +27,10 @@ home = [ "HTML", "REDIRECTS" ]
   newsletterTitle = "September 2018 project update"
   newsletterURL   = "https://medium.com/jaegertracing/september-2018-newsletter-26ae7e8c3f00"
 
+  latest         = "1.7"
```

Comment:
Can we also move the exact latest version from data/downloads to this file? I would rather have those in one top-level place.

Reply:
Sure thing